components/forms/w-field.vue

```diff
@@ -12,60 +12,63 @@
 </template>
 
 <script>
-import { computed } from 'vue'
-import { createValidation } from './validation'
-import { id } from '#util'
-import { modelProps } from 'create-v-model'
+  import { id } from '#util'
+  export const fieldProps = {
+    id,
+    label: String,
+    role: String,
+    invalid: Boolean,
+    hint: String,
+    optional: Boolean,
+    labelLevel: Number,
+    rules: {
+      type: Array,
+      default: () => ([])
+    },
+    ...modelProps(),
+  };
+
+  export default { 
+    inheritAttrs: false,
+    name: 'wField',
+  }
+</script>
 
-export const fieldProps = {
-  id,
-  label: String,
-  role: String,
-  invalid: Boolean,
-  hint: String,
-  optional: Boolean,
-  labelLevel: Number,
-  rules: {
-    type: Array,
-    default: () => ([])
-  },
-  ...modelProps(),
-}
+<script setup>
```

Comment:
I saw the definition of `<script setup>` in [vue docs](https://vuejs.org/api/sfc-script-setup.html) but I couldn't figure out what it changed here. Could you help me understand it?